python/feldera/formats.py

```diff
@@ -0,0 +1,78 @@
+from typing import Optional
+from typing_extensions import Self
+from enum import Enum
+
+
+class UpdateFormat(Enum):
+    """
+    Enum for the update format of the JSON format.
+
+    https://www.feldera.com/docs/api/json/#the-insertdelete-format
+    """
+
+    InsertDelete = 1
+    """
+    InsertDelete: When used, the data is inside a key either "insert" or "delete". This is used to represent changes.
```

Comment:
I have trouble parsing this phrase

Reply:
Me too. I am not sure how to word this properly.